test/routes/routesApi.spec.js

```diff
@@ -1291,6 +1292,32 @@ describe('Le serveur MSS des routes /api/*', () => {
           })
           .catch(done);
       });
+
+      it("utilise l'adaptateur de tracking pour envoyer un événement de connexion", (done) => {
+        let adaptateurAppele = false;
+        testeur.depotDonnees().homologations = () =>
+          Promise.resolve([{ id: '123' }]);
+        testeur.adaptateurTracking().envoieTrackingConnexion = (
+          destinataire,
+          { nombreServices }
```

Comment:
Prenons le paramètre en objet non déstructuré, et faisons le expect en mode `to.eql({…})`